### ISSUE TITLE
Repeated error messages for the same error on version 0.3.5

### ISSUE THREAD

**Issue by lucasdf** [RedCoolBeans/dockerlint#46]
When using version 0.3.5 dockerlint repeats the same erros.

You may reproduce this by linting a Dockefile containing the following content:
```
FROM someimage
ENV	ENV_VARIABLE
```

The output is:
```
ERROR: ENV invalid format ENV_VARIABLE on line 2
ERROR: ENV invalid format ENV_VARIABLE on line 2
ERROR: ENV invalid format ENV_VARIABLE on line 2

ERROR: spec/fixtures/bad/Dockerfile failed.
```

When using dockerlint version 0.3.1 the output is:
```
ERROR: ENV invalid format ENV_VARIABLE on line 2

ERROR: spec/fixtures/bad/Dockerfile failed.
```

**Comment by lucasdf** [comment]
Hi @rnagy 

Now dockerlint does not report the error "ENV invalid format".
